apps/modernization-ui/src/pages/advancedSearch/components/patientSearch/AddressForm.tsx

```diff
@@ -2,7 +2,7 @@ import { Grid } from '@trussworks/react-uswds';
 import { Input } from '../../../../components/FormInputs/Input';
 import { SelectInput } from '../../../../components/FormInputs/SelectInput';
 import { Controller } from 'react-hook-form';
-import { stateList } from '../../../../constant/states';
```

Comment:
Might as well just go ahead and delete the stateList. That way we are sure it wont pop up again

Reply:
@mpeels There's a couple of dead/unlinked pages that reference the stateList--I was hesitant to just delete them, preventing from removing it altogether:
- Mortality.tsx
- SexBirth.tsx
- AddIdentificationModal
- AddressModal

Reply:
Done